Packs/Jira/Integrations/JiraV2/JiraV2_test.py

```diff
@@ -125,6 +125,27 @@ def test_issue_query_command_with_results(mocker):
     assert outputs == QUERY_ISSUE_RESULT
 
 
+def test_issue_query_command_with_custom_fields_with_results(mocker):
+    """
+    Given
+    - Jira issue query command and extraFields parameters
+
+    When
+    - Sending HTTP request and getting one issues from the query
+
+    Then
+    - Verify outputs
+    """
+    from JiraV2 import issue_query_command
+    from test_data.raw_response import QUERY_ISSUE_RESPONSE, FIELDS_RESPONSE
+    from test_data.expected_results import QUERY_ISSUE_RESULT_WITH_CUSTOM_FIELDS
+
+    mocker.patch("JiraV2.get_custom_field_names", return_value=FIELDS_RESPONSE)
```

Comment:
Please mock instead the response from `rest/api/latest/field` instead of the  `get_custom_field_names ` function.
Note that coverage is down by more than 1%.